ctapipe/io/tableio.py

```diff
@@ -168,7 +168,7 @@ def _apply_col_transform(self, table_name, col_name, value):
         return value
 
     @abstractmethod
-    def read(self, table_name: str, container: Container):
+    def read(self, table_name: str, container: Container, prefix=False):
```

Comment:
Isn't this the base method codacy complains about?

Reply:
Yes, now I'm confused. The signatures are the same

Reply:
it makes no sense.  I think I've had it with codacy - it gives to many strange false positives

Reply:
I think the problem is there are two codacy's running: this one comes from Dominik Niese (his username is shown), and it is configured for python2, whereas the one from my account uses python3.  I have no idea how to stop the other one, or to choose which one reports.  It basically complains on any line that is not python2 compatible. Probably we should just kill it and use something else